index.html

```diff
@@ -24,7 +24,7 @@
     <title>em</title>
 
   </head>
-  <body>
+  <body style="padding-top: env(safe-area-inset-top);">
```

Comment:
Let's apply this style in [panda.config.ts](https://github.com/cybersemics/em/blob/82c1f68a31df4086867f3e809c563cdfe2e84ac4/panda.config.ts#L159) for consistency.

Reply:
This is actually unused and I removed it in the latest commit. Its good to go now.